resolvers/webpack/index.js

```diff
@@ -311,6 +311,29 @@ function getResolveSync(configPath, webpackConfig, cwd) {
   return cached.value;
 }
 
+const _evalCache = [];
```

Comment:
is there a reason not to use a null object here, or a Map?

Reply:
Just referred to `getResolveSync()`.
https://github.com/import-js/eslint-plugin-import/blob/67cc79841fc823ad4af2532af2dc6704e4b3b03a/resolvers/webpack/index.js#L294-L312

First I considered to implement it with WeakMap. Reading `getResolveSync()`, I thought it uses array because it is easier to remove old entries. I'm ready to use (Weak)Map here.

Reply:
I changed it to Map. https://github.com/import-js/eslint-plugin-import/pull/3100/commits/c65f9116c6ee7753250fa347036b08615ea4fc2e